sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/internal/SensorContextTester.java

```diff
@@ -145,8 +145,18 @@ public Version getSonarQubeVersion() {
     return sqVersion.get();
   }
 
-  public SensorContextTester setSonarQubeVersion(Version version) {
-    this.sqVersion = new SonarQubeVersion(version);
+  @Override
+  public Version getRuntimeApiVersion() {
+    return sqVersion.get();
+  }
+
+  @Override
+  public boolean isSonarLintRuntime() {
+    return sqVersion.isSonarlintRuntime();
+  }
+
+  public SensorContextTester setRuntime(Version version, boolean isSonarLint) {
+    this.sqVersion = new SonarQubeVersion(version, isSonarLint);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "SonarQubeVersion"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)